# Lab | Inferential statistics


### Instructions

1. It is assumed that the mean systolic blood pressure is `μ = 120 mm Hg`. In the Honolulu Heart Study, a sample of `n = 100` people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

   - Set up the hypothesis test.
   - Write down all the steps followed for setting up the test.
   - Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

2. If you finished the previous question, please go through the code for `principal_component_analysis_example` provided in the `files_for_lab` folder .






## test set up

1. Nullhypothesis (status quo, suspect is not guilty) $H_0: \mu = 120$, and the resulting mean ($130.1 pounds$) was due to a sampling error.
2. Alternative hypothesis: $H_1: \mu > 120pounds$.
3. level of significance: $\alpha$ = 5%
4. Calculate a **Z-score**
5. Decide to reject or accept the null hypothesis

In [15]:
import math 

sample_mean = 130.1
pop_mean = 120
sample_std = 21.21
n = 100 #  sample size
z_score = (sample_mean - pop_mean) / (sample_std)
z_score


0.4761904761904759

sample mean is 0.476 st dev away from the pop mean

as derived from z-table, p-value is 32%.

the probability to measure the diff in means of (130.1 - 120) is 32%, which is higher then the signif level. So we accept the null hypothesis.

In [16]:
# derive p-value in Python

import scipy.stats

p = scipy.stats.norm.sf(abs(z_score))

p

0.316969341940673